### ISSUE TITLE
SA1610 inheritdoc analysis does not consider overrides

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1945]
The following code should report SA1610, but no warning is reported.

```csharp
/// <value>
/// </value>
/// <inheritdoc/>
public string Property => "P";
```
